ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCUI-nonprod.json

```diff
@@ -79,5 +79,26 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NOTIFY_LIP_DEFENDANT_RESPONSE_SUBMISSION",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-systemupdate"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
```

Comment:
Is it better to following other roles in the nonprod.json?
Like the following
"APP-SOL-SPEC-PROFILE",
"caseworker-civil-admin",
"RES-SOL-ONE-SPEC-PROFILE",
"RES-SOL-TWO-SPEC-PROFILE"

Reply:
ah i see what you mean. will need to change this for the applicant email as well - it looks the same as mine

Reply:
done and also fixed for the applicant (or claimant email)